.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: node_js
+node_js:
+- "4.1"
+install:
+- npm install gitbook-cli -g
+script:
+- gitbook install
+- gitbook build
+skip_cleanup: true
+deploy:
+  provider: pages
+  skip-cleanup: true
```

Comment:
Does this need `skip-cleanup: true` twice?

Reply:
Fixed.